app/controllers/users_controller.rb

```diff
@@ -16,4 +16,8 @@ def show
   def user_find
     @user = User.find_by(id: params[:id])
   end
+
+  def user_params
+    params.permit(:name, :description, :avatar, :avatar_url, :gender, :birthday, :phone)
+  end
```

Comment:
Where is this method used?

Reply:
This is already added in a branch with an avatar download for users, it's necessary to remove from this pull request. I've already done it.